src/PackageHelper.cs

```diff
@@ -119,17 +121,22 @@ public void WriteError(string error, params object[] parameters)
         /// <summary>
         ///  basic http client with Bearer token setup.
         /// </summary>
-        public HttpClient GetClientBase(string apiKey)
+        public HttpClient GetClientBase(string url, string apiKey, int memberId, int projectId)
         {
-            var client = new HttpClient();
+            var baseUrl = AuthConstants.BaseUrl;
+            var client = new HttpClient {BaseAddress = new Uri(baseUrl)};
```

Comment:
HttpClient instances should not be re-created each time, see https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/ ... but in netcore which you are using you should be using IHttpClientFactory https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests